shared/js/dialogs.js

```diff
@@ -46,10 +46,17 @@ var Dialogs = {
         confirmButton.classList.remove('danger');
       }
 
+      if (options.bodyClass) {
```

Comment:
I'm always nervous about changes to shared code, but this one is clearly opt-in, so it seems safe. I would like a comment before this line:

// If the caller specified a bodyClass, add that to the classList of the body element.
// This can be used or accessibility, e.g., to hide the main document while a dialog is shown